modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,124 @@ If you have installed the {gitops-title} Operator in the default namespace, run
+<3> Specifies the name of the volume to be mounted, which must match the name of the `volume` defined in volumes.
+<4> Specifies the path inside the container where the volume will be mounted.
+
+* With this update, users can add custom labels and annotations to the server, repo-server, applicationset-controller, and application-controller pods. This enhancement improves observability for external systems that depend on resource labels or annotations for monitoring purposes. The following fields are introduced in the Argo CD CR to support this functionality:
+
+** `.spec.server.labels`
+** `.spec.server.annotations`
+** `.spec.repo.labels`
+** `.spec.repo.annotations`
+** `.spec.controller.labels`
+** `.spec.controller.annotations`
+** `.spec. applicationSet.labels`
+** `.spec. applicationSet.annotations` 
+
+For more information, see link:https://argocd-operator.readthedocs.io/en/latest/usage/custom_labels_annotations/[Custom Annotations and Labels]. link:https://issues.redhat.com/browse/GITOPS-5674[GITOPS-5674]
+
+* Before this update, users could not configure some replicas for the Argo Rollouts controller by using the `RolloutManager` custom resource (CR). With this update, high availability (HA) support is introduced by adding a new `.spec.ha` field to the `RolloutManager` CR.
++
+When you enable high availability, the operator automatically sets the number of replicas to 2, the leader election is activated, and default anti-affinity rules are applied to ensure redundancy and reliability. link:https://issues.redhat.com/browse/GITOPS-5622[GITOPS-5622]
+
+* With this update, the Argo CD multi-source application functionality in {gitops-shortname} Operator is promoted from Technology Preview (TP) to General Availability (GA). Use this feature to define and manage applications with multiple source repositories. link:https://issues.redhat.com/browse/GITOPS-5511[GITOPS-5511]
+
+* With this update, you can now use Kubernetes TLS-type secrets to configure custom TLS certificates for Routes in the Argo CD CR. Previously, TLS data had to be embedded directly in the Argo CD CR as plain text, which could pose security risks due to the sensitive nature of TLS information.
++
+To enhance security, you must use Kubernetes secrets to manage TLS data. The following fields are introduced in the Argo CD CR to reference Kubernetes secrets of type `kubernetes.io/tls` containing TLS data:
+
+** `.spec.server.route.tls.externalCertificate`
+** `.spec.applicationSet.webhookServer.route.tls.externalCertificate`
+** `.spec.prometheus.route.tls.externalCertificate`
+** `.spec.grafana.route.tls.externalCertificate`
+
+For more information about configuring a route for custom TLS certificate, see examples in link:https://argocd-operator.readthedocs.io/en/latest/usage/routes/#custom-tls-certificates[Custom TLS certificates for Routes]. link:https://issues.redhat.com/browse/GITOPS-5232[GITOPS-5232]
```

Comment:
Same suggestion that I made earlier (though test to see if it works):
```suggestion
+
--
** `.spec.server.route.tls.externalCertificate`
** `.spec.applicationSet.webhookServer.route.tls.externalCertificate`
** `.spec.prometheus.route.tls.externalCertificate`
** `.spec.grafana.route.tls.externalCertificate`
--
+
For more information about configuring a route for custom TLS certificate, see examples in link:https://argocd-operator.readthedocs.io/en/latest/usage/routes/#custom-tls-certificates[Custom TLS certificates for Routes]. link:https://issues.redhat.com/browse/GITOPS-5232[GITOPS-5232]
```

Reply:
Fixed.